res/openvpn-gui-res-it.rc

```diff
@@ -2,6 +2,7 @@
  *  OpenVPN-GUI -- A Windows GUI for OpenVPN.
  *
  *  Copyright (C) 2004 Mathias Sundman <mathias@nilings.se>
+ *                2009 Heiko Hund <heikoh@users.sf.net>
```

Comment:
why is this here? I am not sure how it is realated to the translation

Reply:
> 
> 
> @selvanair, I have checked the localization file against the English version and all strings are now up to date. I have added new strings, updated a few (including eg. copyright headers) and corrected a few more inconsistencies.
> 
> I have also added keyboard shortcut support ("&" tokens) where missing.

Please refer to the November 22 and 23 entries. As suggested by @selvanair, after my first PR, I checked the whole localization file against the English version and completed the job with the new fields. That included updating the copyright headers to match the ones on the English version.

Reply:
But copyright indicates authorship of the file -- no point in adding someone else's name in a file you are editing :) I would leave it out though we have not been very strict/correct (IMO) about copyright year updates and attribution.. What I do is add/update my name and year when a substantial contribution is added to a file and never add or change anyone else's name or year.

Reply:
My mistake, I thought it was a project-wise copyright notice. I will revert this update.